ci/token.sh

```diff
@@ -13,8 +14,12 @@ cd "$(dirname "$0")/.."
 ./do.sh test token
 cc token/inc/token.h -o token/target/token.gch
 
-cd "$(dirname "$0")/../token/js"
+# Rust API
+cd "$(dirname "$0")/../token"
+cargo test
```

Comment:
this should be the same as `./do.sh build-native token`

Reply:
ah, ok.  removed.